.travis/docker-wait-healthy

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
should create a repository such as `libero/scripts` or similar to pull these from, with a series of `wget`. This is going to appear in 3 repositories very soon

Reply:
Though I've always avoided them, maybe a real use case for submodules? Especially given https://dependabot.com/submodules/.

Reply:
Would be a first-class solution, it makes more complex to deploy the source but we are not doing that favoring an artifact like a Docker image instead.

Reply:
Will investigate separately not to hold up this.